physics/discrete_trajectory.hpp

```diff
@@ -108,9 +108,9 @@ class DiscreteTrajectory : public Forkable<DiscreteTrajectory<Frame>,
   // trajectory.
   not_null<DiscreteTrajectory<Frame>*> NewForkAtLast();
 
-  // The first point of |fork| is removed from |fork| and appended (using
-  // Append) to this trajectory.  Then |fork| is made a fork of this trajectory
-  // at the newly-inserted point.  |fork| must be a non-empty root.
+  // Changes |fork| to become a fork of this trajectory at the end of this
+  // trajectory.  |fork| must be a non-empty root.  It may or may not have a
+  // point at the same time as the end of this trajectory.
```

Comment:
Clarify the requirements on `fork` that were implicit in `using Append`, e.g., it must not have points before the end of the trajectory, etc.

Clarify the behaviour with respect to the possible common point: which one is kept, etc.

Reply:
Done.